main/guides/js-programming/agoric-js-overview.md

```diff
@@ -107,13 +107,10 @@ additions and not part of JavaScript.
 Most Node.js-specific [global objects](https://nodejs.org/dist/latest-v14.x/docs/api/globals.html) 
 are unavailable within a vat including:
 
-* `queueMicrotask`
+* `queueMicrotask`: You can 
+  replace `queueMicrotask(fn)` with `Promise.resolve().then(_ => fn())`.
```

Comment:
I had just copy-pasted the code from another place, and technically there would be a difference if `fn` is declared as such:
```js
function fn(...args) {
  console.log(args.length);
}
```

That said, if you think it makes the example more clear, happy to change.

Reply:
Interesting example! Since the one argument would be a single `undefined` the shorter code is mostly-harmlessly different that the code it replaces. I think we should use the shorter code.

Reply:
Just saw your keybase chat note linking to https://jakearchibald.com/2021/function-callback-risks/ . Good point! Ok, the original is fine. No change suggested.